src/_guides/language/language-tour.md

```diff
@@ -1422,17 +1419,17 @@ list.forEach((item) {
 });
 {% endprettify %}
 
-Click the run button {% asset red-run.png alt="" %} to execute the code.
+Click **Run** to execute the code.
 
 {% comment %}
 https://gist.github.com/chalin/5d70bc1889d055c7a18d35d77874af88
 {{site.dartpad}}/5d70bc1889d055c7a18d35d77874af88
 {% endcomment %}
 
 <iframe
-src="{{site.custom.dartpadx.embed-inline-prefix}}?id=5d70bc1889d055c7a18d35d77874af88"
+src="{{site.custom.dartpadx.embed-inline-prefix}}?id=5d70bc1889d055c7a18d35d77874af88&split=60"
     width="100%"
-    height="250px"
+    height="400px"
```

Comment:
This seems taller than necessary, but I don't think that's a problem. This way they get space to play...